assets/src/dashboard/icons/playArrow.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="11" height="14" viewBox="0 0 11 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M0 0V14L11 7L0 0Z" fill="black"/>
+<path d="M0 0V14L11 7L0 0Z" fill="currentColor"/>
```

Comment:
Did you experience issues with this otherwise? I thought our SVGO implementation automatically changes these attributes to use `currentColor`.

Reply:
Oh I didn't even realize it did that. Yes, we have had to change it to `currentColor` otherwise it uses the color listed.